effect/src/main/scala/shims/effect/instances/TaskInstances.scala

```diff
@@ -16,10 +16,11 @@
 
 package shims.effect.instances
 
-import cats.StackSafeMonad
+import cats._
```

Comment:
I would really much prefer specific imports here, rather than using `_`. It's better documentation and a more restricted implicit scope.

Reply:
There aren't any implicits in `cats._` but sure, I will change the import.